tests/test_loadDiaCatalogs.py

```diff
@@ -390,6 +395,25 @@ def testGetPixelRanges(self):
             self.assertEqual(normPix, testPix)
             self.assertEqual(flipPix, testPix)
 
+    def test_apdbSchema(self):
```

Comment:
This only tests if the default schema for adpb and the functors run by default match. Also, it is safe to have extra columns in the apdb schema not filled by the DiaSource functor file. The main testing I've been doing to make sure things work has been running ap_verify as an integration test. That only just failed, as we've discovered, as sqlite is not case sensitive in it's column names but Postgres is. We can leave these here for now, but I will point out that it does not protect against mismatches in the schema files.

Reply:
How does this not protect against mismatches in the schema files? This test ensures that everything is case-consistent as well.

Reply:
The schemas can be set at run time. This only tests for the default ones. It doesn't even pull in all columns that are currently being used and written to the apdb.